src/glwidget.cpp

```diff
@@ -368,7 +372,13 @@ void GLWidget::paintGL()
             glDepthFunc(GL_LESS);
             glClearColor(0.0, 0.0, 0.0, 1.0);
             glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
-            renderText(width() / 3, height() / 2, "no map or no valid position on map", QFont("Bitstream Vera Sans Mono", 30, QFont::Normal));
+
+            QPainter painter(this);
+            painter.setPen(Qt::white);
```

Comment:
Same here

Reply:
Added your suggestions and Antialiasing. For me it looks even better then the old 3d mapper with better performance :)

Reply:
Yeah it does look better! And by performance you mean the overlay labels?

Reply:
Yes, As mentioned in https://github.com/Mudlet/Mudlet/issues/3321 the old mapper also created some performance issues especially with big gui (but don't know if the difference is noticeable).